.travis.yml

```diff
@@ -249,17 +279,17 @@ script:
   - |
     if [[ "$SKIP_TEST_RUN" != "true" ]]; then
       echo "travis_wait ../scripts/run-ci-tests.sh ${CI_TEST_FLAGS}"
-      travis_wait ../scripts/run-ci-tests.sh ${CI_TEST_FLAGS}
+      travis_wait $SHELL_CMD ../scripts/run-ci-tests.sh ${CI_TEST_FLAGS}
```

Comment:
are we only using the scripts on the MSYS build?

Reply:
Scripts are used for all of the builds. The commands used for builds/tests are the same other than using a SHELL_CMD wrapper (if defined) and adding the MSYS bin folder to the PATH.

Reply:
This entire script section in the yaml could have been put in a .sh file, but I think that would make Travis not display as much information on how long subcommands take to run.